enderio-base/src/main/java/com/enderio/base/common/enchantments/SoulBoundHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.enderio.base.common.enchantments;
```

Comment:
I'd rather this file just not be here right now; I don't like having an empty file with old 1.12 code in